src/main/java/com/medallia/word2vec/Word2VecModel.java

```diff
@@ -137,13 +138,13 @@ public static Word2VecModel fromBinFile(File file, ByteOrder byteOrder, Profilin
 			// every gigabyte. To calculate offsets correctly, we have to keep track how many gigabytes
 			// we've already skipped. That's what this is for.
 
-			StringBuilder sb = new StringBuilder();
-			char c = (char) buffer.get();
+			List<Byte> list = new ArrayList<Byte>();
+			byte c = buffer.get();
 			while (c != '\n') {
```

Comment:
You are using a hard coded line separator here. Do you have a specific reason not to use [System.lineSeparator()](https://docs.oracle.com/javase/8/docs/api/java/lang/System.html#lineSeparator--) instead?